torchgeo/datasets/seasonet.py

```diff
@@ -0,0 +1,456 @@
+
+    * images are 16-bit GeoTiffs, split into seperate files based on resolution
+    * images include 12 spectral bands with 10, 20 and 60 m per pixel resolutions
+    * masks are single-channel 8-bit GeoTiffs
+
+    Dataset classes:
+
+    0. Continuous urban fabric
+    1. Discontinuous urban fabric
+    2. Industrial or commercial units
+    3. Road and rail networks and associated land
+    4. Port areas
+    5. Airports
+    6. Mineral extraction sites
+    7. Dump sites
+    8. Construction sites
+    9. Green urban areas
+    10. Sport and leisure facilities
+    11. Non-irrigated arable land
+    12. Vineyards
+    13. Fruit trees and berry plantations
+    14. Pastures
+    15. Broad-leaved forest
+    16. Coniferous forest
+    17. Mixed forest
+    18. Natural grasslands
+    19. Moors and heathland
+    20. Transitional woodland/shrub
+    21. Beaches, dunes, sands
+    22. Bare rock
+    23. Sparsely vegetated areas
+    24. Inland marshes
+    25. Peat bogs
+    26. Salt marshes
+    27. Intertidal flats
+    28. Water courses
+    29. Water bodies
+    30. Coastal lagoons
+    31. Estuaries
+    32. Sea and ocean
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://doi.org/10.1109/IGARSS46834.2022.9884079
+    """
+
+    url = "https://zenodo.org/api/records/5850306"
+    file_names = [
+        "spring.zip",
+        "summer.zip",
+        "fall.zip",
+        "winter.zip",
+        "snow.zip",
+        "splits.zip",
+        "meta.csv",
+    ]
+    extracted_names = [
+        "spring",
+        "summer",
+        "fall",
+        "winter",
+        "snow",
+        "splits",
+        "meta.csv",
+    ]
+    classes = [
+        "Continuous urban fabric",
+        "Discontinuous urban fabric",
+        "Industrial or commercial units",
+        "Road and rail networks and associated land",
+        "Port areas",
+        "Airports",
+        "Mineral extraction sites",
+        "Dump sites",
+        "Construction sites",
+        "Green urban areas",
+        "Sport and leisure facilities",
+        "Non-irrigated arable land",
+        "Vineyards",
+        "Fruit trees and berry plantations",
+        "Pastures",
+        "Broad-leaved forest",
+        "Coniferous forest",
+        "Mixed forest",
+        "Natural grasslands",
+        "Moors and heathland",
+        "Transitional woodland/shrub",
+        "Beaches, dunes, sands",
+        "Bare rock",
+        "Sparsely vegetated areas",
+        "Inland marshes",
+        "Peat bogs",
+        "Salt marshes",
+        "Intertidal flats",
+        "Water courses",
+        "Water bodies",
+        "Coastal lagoons",
+        "Estuaries",
+        "Sea and ocean",
+    ]
+    all_seasons = {"Spring", "Summer", "Fall", "Winter", "Snow"}
+    all_bands = ("10m_RGB", "10m_IR", "20m", "60m")
+    band_nums = {"10m_RGB": 3, "10m_IR": 1, "20m": 6, "60m": 2}
+    splits = ["train", "val", "test"]
+    colormap = [
```

Comment:
Let's name this `cmap` and use the same dict structure as CDL and NLCD.

Reply:
Done